components/brave_new_tab_ui/components/default/braveToday/customize/SourcesListEntry.tsx

```diff
@@ -86,7 +87,7 @@ export function ChannelListEntry (props: { channelId: string }) {
   const { setSubscribed } = useChannelSubscribed(props.channelId)
 
   return <Container direction="row" justify='space-between' align='center'>
-    <ChannelNameText>{props.channelId}</ChannelNameText>
+    <ChannelNameText>{getTranslatedChannelName(props.channelId)}</ChannelNameText>
```

Comment:
Q. Can we rename props.channelId to props.channelName? The id seems to be used as name as well but I think renaming would be good 😄

Reply:
Renamed! Mind taking another look?